cmake/modules/BuildJaeger.cmake

```diff
@@ -24,7 +24,7 @@ function(set_library_properties_for_external_project _target _lib)
     INTERFACE_LINK_LIBRARIES "${_libpath}"
     IMPORTED_LINK_INTERFACE_LANGUAGES "CXX"
     IMPORTED_LOCATION "${_libpath}"
-    INTERFACE_INCLUDE_DIRECTORIES "${_includepath}")
+    INTERFACE_SYSTEM_INCLUDE_DIRECTORIES "${_includepath}")
```

Comment:
@ideepika what build warning are you trying to suppress ? could you paste a sample of them?